datalad/local/tests/test_add_readme.py

```diff
@@ -69,8 +69,75 @@ def test_add_readme(path):
 
 This is a DataLad dataset (id: {id}).
 
-For more information on DataLad and on how to work with its datasets,
-see the DataLad documentation at: http://handbook.datalad.org
+## DataLad datasets and how to use them
```

Comment:
Dear @loj , I know that you are just extending what was already in the test here, but especially with such expansion of the README text, I think we should move away from a test to be a test of a developer to be able to copy-paste text from the code into the test and to the test of expectations.  Otherwise - any typo or wording fix in the code, would require identical fix here, just wasting effort IMHO for no gain.

I would have 
- read the file into a variable
- made previous comparison into ok_startswith (just to be less intrusive)
- tested for inclusion of critical pieces you have (e.g. mentioning of the handbook, having "Get the dataset" section etc)
- tested that each line is not exceeding e.g. 160 characters (to account for `[url](s)`)
- or even, if pypandoc is installed, tried to convert that README.md into e.g. `rst` thus verifying that it is correct markdown, e.g. with smth like `python -c 'import pypandoc; print(pypandoc.convert_text(open("README.md").read(), "rst", "markdown"))'` what I did in CLI.  I think it would be a very weak validation (I have broke README.md markdown and pandoc still converted :-/) but hopefully would still catch some very errorneous cases.

But may be it is just me, and other @datalad/developers would disagree and would prefer a verbatim cut/paste?

Reply:
*waves: blame me!* 
sorry, this was me, @yarikoptic

Reply:
I have improved the commit based on your edits. thx.